front/app/components/Form/Components/Controls/LinearScaleControl.tsx

```diff
@@ -126,6 +123,20 @@ const LinearScaleControl = ({
     event.preventDefault();
   };
 
+  // Put all labels from the UI Schema in an array
+  const labelsFromSchema = Array.from({ length: maximum }).map((_, index) => {
+    if (uischema?.options?.[`linear_scale_label${index + 1}`]) {
+      return uischema.options[`linear_scale_label${index + 1}`];
+    }
+  });
+
+  // Get an array of the middle value labels so we can determine how to show them in the UI
+  const middleValueLabels = labelsFromSchema
+    .map((label) => label) // First copy the array so we don't mutate the original
+    .splice(1, labelsFromSchema.length - 2); // Get only the middle values
```

Comment:
We can do away with the map and use slice to create a new array starting with the second element to the second to last element
```suggestion
  const middleValueLabels = labelsFromSchema.slice(1, -1);
```